cmd/publish.go

```diff
@@ -56,9 +56,15 @@ dapr publish --enable-domain-socket --publish-app-id myapp --pubsub target --top
 
 		client := standalone.NewClient()
 		// TODO(@daixiang0): add Windows support
-		if runtime.GOOS == "windows" && publishSocket != "" {
-			print.FailureStatusEvent(os.Stderr, "unix-domain-socket option still does not support Windows!")
-			os.Exit(1)
+		if unixDomainSocket != "" {
+			if runtime.GOOS == "windows" && invokeSocket != "" {
```

Comment:
should be publishsocket instead of invokesocket?

Reply:
Yes. Corrected